### PR TITLE
GH-3742 add Install Quilt button

### LINKED ISSUE #3742: Add Quilt Mod Loader.
to here
image

https://quiltmc.org/faq/
It's a Fabric fork.

### PR DESCRIPTION
**Pull request by kb-1000**
Quilt beta has been released now.

Requires MultiMC/meta#4.
Closes #3742.

### CODE REVIEW

File: application/pages/instance/VersionPage.cpp [hunk 1]
```diff
@@ -461,6 +462,33 @@ void VersionPage::on_actionInstall_Fabric_triggered()
...
+void VersionPage::on_actionInstall_Quilt_triggered()
+{
+    auto vlist = ENV.metadataIndex()->get("org.quiltmc.quilt-loader");
+    if(!vlist)
+    {
+        return;
+    }
+    VersionSelectDialog vselect(vlist.get(), tr("Select Quilt Loader version"), this);
+    vselect.setEmptyString(tr("No Quilt Loader versions are currently available."));
+    vselect.setEmptyErrorString(tr("Couldn't load or download the Fabric Loader version lists!"));
```
Comment (ImperatorStorm) [inline application/pages/instance/VersionPage.cpp:474]: shouldn't this be "Quilt Loader"?

**Review by ImperatorStorm — CHANGES_REQUESTED** [review]

File: application/pages/instance/VersionPage.cpp (hunk 1 above)
Comment (kb-1000) [reply application/pages/instance/VersionPage.cpp]: Yes it should, fixed

**Comment by peterix** [comment]
Hm. This is a bit premature and pretty small compared to the changes needed in meta and elsewhere.

**Comment by kb-1000** [comment]
> This is a bit premature

There's a reason why this was marked as a draft PR.

**Comment by glowiak** [comment]
We are four parallel universes ahead of you

**Pull request merged**